spec/requester/get_spec.rb

```diff
@@ -26,7 +26,7 @@
       end
 
       it "should request properly with params in the body" do
-        @stubs.send(:new_stub, :get, '/test', "body data") {
+        @stubs.send(:new_stub, :get, '/test', {}, "body data") {
```

Comment:
Avoid using {...} for multi-line blocks.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.